src/tests/t_y2038.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/python
+from k5test import *
+
+# These tests will become much less important after the y2038 boundary
+# has elapsed, and may start exhibiting problems around the year 2075.
```

Comment:
I don't think we need a warning for y2038 elapsing.  After that point, testing positive-to-negative timestamp transition scenarios will be moot.

I also don't think we need a specific warning for the early y2106 failure in the test suite.  If the project still has enough life in it in 2075 to successfully do y2106 remediation, people will probably have started thinking it by then, or at least have revisited this test script.